### DISCUSSION TITLE
Hypixel bypass

### DISCUSSION THREAD

**Discussion by Brylsonek** [ORelio/Minecraft-Console-Client#2243] [General]
can't join hypixel
![Zrzut ekranu 2022-10-05 200641](https://user-images.githubusercontent.com/74657176/194130916-a6084003-887f-4dfe-b928-b13faea2463a.png)

**Comment by milutinke** [comment]
They don't allow versions above `1.19`.
Go to `MinecraftClient.ini` and change `mcversion` from `auto` to your desired version.
Example: `mcversion=1.18.2`

Just to give you a warning, MCC can get your account banned if spotted by admins, also Anti Cheat could detect bot movement and ban you, has happened to few people before.

**Reply by Brylsonek** [reply]
Still not work, any other ideas?

**Comment by breadbyte** [comment]
Replace the `brandinfo` in settings to vanilla.

**Comment by Brylsonek** [comment]
Thanks, problem has been solved
